src/main/java/frc/robot/commands/ChargeStationBalance.java

```diff
@@ -7,11 +7,18 @@
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import edu.wpi.first.wpilibj.Timer;
-import frc.robot.Constants;
 import frc.robot.subsystems.DriveTrain;
+import edu.wpi.first.math.geometry.Rotation2d;
 
 public class ChargeStationBalance extends CommandBase {
 
+  //private final Rotation2d m_rampAngle = Rotation2d.fromDegrees(-10); //useful if we are going to automate going on the ramp
+  private final Rotation2d m_balancedError = Rotation2d.fromDegrees(2.5); //error for what counts as balanced
+  private final double m_balancedDegrees = 0;
+  private final double m_balanceKP = 0.02; //change to control how fast robot drives during balancing
+  private final double m_maxMPS = 0.75;
```

Comment:
if a variable is final, then the value won't change again. Therefore it usually is a constant, so the name should be all capitalized letters